x-pack/plugins/endpoint/common/types.ts

```diff
@@ -122,6 +145,23 @@ export interface EndpointMetadata {
   };
 }
 
+export interface ESTotal {
+  value: number;
+  relation: string;
+}
+
+export interface AlertDataWrapper {
+  _id: string;
+  _source: AlertData;
+}
+
+export type AlertHits = AlertDataWrapper[];
```

Comment:
This is still needed for some things, as `GetResponse<T>` requires some fields that are not always present (e.g. `found: boolean`.